src/Ledger/Transaction.lagda

```diff
@@ -35,11 +35,15 @@ the transaction body are:
   \item The size and the hash of the serialized form of the transaction that was included in the block.
 \end{itemize}
 
+Finally, $\fun{txid}$ computes the transaction id of a given transaction.
```

Comment:
This also undoes cleanup: I recently removed this paragraph, since it's wrong (it used to be true).